File: BackEnd/src/main/java/com/example/backend/model/MovieDetail.java
package com.example.backend.model;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 * packageName : com.example.dscine.model
 * fileName : MovieDetail
 * author : Moon
 * date : 2022-12-28-028
 * description :
 * ===========================================================
 * DATE            AUTHOR             NOTE
 * —————————————————————————————
 * 2022-12-28-028         Moon          최초 생성
 */
@Data
@Entity
@ToString
@NoArgsConstructor
@Table(name = "DETAIL_MOVIE")
public class MovieDetail
{
//    private Integer id;

    @Id
    private String moviecd;

    private String movienm;

    private String movienmen;

    private String movienmog;

    private String prdtyear;

    private String showtm;

    private String opendt;

    private String prdtstatnm;

    private String nationnm;

    private String genrenm;

    private String directors;

    private String watchgradenm;

    private String actor;

    private String cast;

    private Double raiting;

    private String posterurln;

    private String posterurlkm;

    private String imgurl;

    private String utubeurl;

    private String plot;
}

File: frontEnd_Original/src/model/Moviedetail.js
export default class Moviedetail {
    // js 생성자 : constructor(매개변수...) {}
    constructor(
        id,
        moviecd,
        movienm,
        movienmen,
        movienmog,
        prdtyear,
        showtm,
        opendt,
        prdtstatnm,
        nationnm,
        genrenm,
        directors,
        watchgradenm,
        actor,
        cast,
        raiting,
        posterurln,
        posterurlkm,
        imgurl,
        utubeurl,
        plot
    ) {
        this.id = id;
        this.moviec = moviecd;
        this.movienm = movienm;
        this.movienmen = movienmen;
        this.movienmog = movienmog;
        this.prdtyear = prdtyear;
        this.showtm = showtm;
        this.opendt = opendt;
        this.prdtstatnm = prdtstatnm;
        this.nationnm = nationnm;
        this.genrenm = genrenm;
        this.directors = directors;
        this.watchgradenm = watchgradenm;
        this.director = actor;
        this.cast = cast;
        this.raiting = raiting;
        // this.posterurl = posterurln;
        this.posterurln = posterurln;
        this.posterurlkm = posterurlkm;
        this.imgurl = imgurl;
        this.utubeurl = utubeurl;
        this.plot = plot;
    }
}